source/collections/Vector2D.ooc

```diff
@@ -38,6 +40,58 @@ Vector2D: class <T> {
 	_elementPosition: func (row, column: Int, columnCount := this columnCapacity) -> Int {
```

Comment:
It's still going to be confusing, if `Vector` has capacity and `Vector2D` has count, and I'm not sure what the correct terminology would be in `Vector` then. Still, yes, go ahead and change to `count` here and leave `Vector` as is for now.